docs/seo.md

```diff
@@ -0,0 +1,57 @@
+# SEO Metadata
+
+SnapWP exports two main functions to add meta tags for SEO:
+
+-   `getLayoutMetadata`: Gets SEO data for the site.
+-   `getPageMetadata`: Gets SEO data related to the template. Adds update icon
+
+## Registering a new plugin
+
+The SEO functionality can be extended by registering new plugins. Below is an example of updated root layout.
+
+```typescript
+import { RootLayout } from '@snapwp/next';
+import { getLayoutMetadata, Seo } from '@snapwp/next/seo';
+import type { Metadata } from 'next';
+import type { PropsWithChildren } from 'react';
+
+// Register a plugin before calling function `getLayoutMetadata`
```

Comment:
Registration should happen for now in RootLayout?

Reply:
Yes, it's not connected to the config right now.